test/run-tests

```diff
@@ -101,7 +101,7 @@ def run(*argv, env=None, check=True):
     print("+ " + envrepr + " ".join(shlex.quote(a) for a in argv))
```

Comment:
It might make sense to add the new working directory to the debug output, maybe:

```
if cwd:
    cwdrepr = " cd {} ".format(shlex.quote(cwd))
else:
    cwdrepr = ""
```

and then use 

```
print("+ " + envrepr + cwdrepr + " ".join(shlex.quote(a) for a in argv))
```

Reply:
@yontalcar did you see my comment? Do you agree with this change?